src/plugins/Plugin.js

```diff
@@ -95,6 +103,10 @@ module.exports = class Plugin {
     }
   }
 
+  focus () {
+    return
+  }
```

Comment:
Do we need `focus` for something after all?

Reply:
no, it's not being used anywhere. will remove, thanks!